bugbug/similarity.py

```diff
@@ -169,6 +173,17 @@ def evaluation(self):
                 num_hits = 0
                 queries += 1
                 similar_bugs = self.get_similar_bugs(bug)[:10]
+                if end_to_end:
+                    sim_bugs = [
+                        bug for bug in bugzilla.get_bugs() if bug["id"] in similar_bugs
+                    ]
+                    bug_couples = [(bug, sim_bugs[bug_id]) for bug_id in sim_bugs]
+                    probs = duplicatemodel.classify(bug_couples, probabilities=True)
+                    similar_bugs = [
+                        similar_bugs[idx]
+                        for idx, prob in enumerate(probs)
+                        if prob[1] > 0.8
```

Comment:
The 0.8 value should be configurable with a parameter (add the parameter to the constructor and save it in an attribute)